src/main/java/bio/terra/app/controller/DatasetsApiController.java

```diff
@@ -208,7 +208,7 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
 
   @Override
   public ResponseEntity<JobModel> removeDatasetAssetSpecifications(
-      @PathVariable("id") UUID id, @PathVariable("assetId") String assetId) {
+      @PathVariable("id") UUID id, @PathVariable("assetid") String assetId) {
```

Comment:
This is the wrong way to fix this problem. We shouldn't be using spring-web annotations at all in our controller subclasses, since the subclass copies aren't re-generated when the openapi yaml is changed. This is exactly why duplicating these annotations isn't good.
```suggestion
      UUID id, String assetId) {
```

Reply:
I wasn't sure if they still made it through if you don't annotate them in the implementing method.  Might have been an older Spring behavior

Reply:
Made a ticket to correct this. Won't do it now. https://broadworkbench.atlassian.net/browse/DR-2442

Reply:
> I wasn't sure if they still made it through if you don't annotate them in the implementing method. Might have been an older Spring behavior

I don't know how long it's worked this way, but the correctness can be verified via a mock MVC test on the endpoint (in addition to e2e tests of course). It would be great if we had at least one mock MVC test per endpoint.